packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -0,0 +1,181 @@
+import { Accessibility, datepickerBehavior, DatepickerBehaviorProps } from '@fluentui/accessibility';
+import { getElementType, useAccessibility, useStyles, useTelemetry, useUnhandledProps } from '@fluentui/react-bindings';
+import { Ref } from '@fluentui/react-component-ref';
+import * as _ from 'lodash';
+import * as React from 'react';
+// @ts-ignore
+import { ThemeContext } from 'react-fela';
+import {
+  FluentComponentStaticProps,
+  ProviderContextPrepared,
+  WithAsProp,
+  withSafeTypeForAs,
+  ShorthandRenderFunction,
+} from '../../types';
+import { commonPropTypes, createShorthandFactory, UIComponentProps } from '../../utils';
+// import DatepickerInput from './DatepickerInput';
+// import DatepickerCalendar from './DatepickerCalendar';
+import Input from '../Input/Input';
+import Button from '../Button/Button';
+import { CalendarIcon } from '@fluentui/react-icons-northstar';
+import Popup from '../Popup/Popup';
+import {
+  DayOfWeek,
+  IDayGridOptions,
+  FirstWeekOfYear,
+  DateRangeType,
+  IDateGridStrings,
+  formatMonthDayYear,
+  IDateFormatting,
+} from '@fluentui/date-time-utilities';
+import DatepickerCalendar from './DatepickerCalendar';
+
+const DEFAULT_STRINGS: IDateGridStrings = {
+  months: [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ],
+  shortMonths: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
+  days: ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'],
+  shortDays: ['S', 'M', 'T', 'W', 'T', 'F', 'S'],
+};
+
+export interface DatepickerProps extends IDateGridStrings, IDayGridOptions, IDateFormatting, UIComponentProps {
```

Comment:
The DatepickerProps have been chosen based on the [Datepicker design doc](https://github.com/microsoft/fluentui/blob/master/specs/Datepicker.md). 

Some properties contain a small change in the name (based on what was previously in fabric-ui).

Notable breakdown:
* IDayGridOptions contains extra properties
  * firstWeekOfYear: FirstWeekOfYear;
  * daysToSelectInDayView?: number;
  * today?: Date;
  * showWeekNumbers?: boolean;
  * weeksToShow?: number;
  * workWeekDays?: DayOfWeek[];
  * navigatedDate: Date; 

* General functioning of Datepicker
  * disabled	boolean	Datepicker can show it is currently unable to be interacted with.
  * isRequired	boolean	Datepicker can show it's input is required to be filled.
  * onChange	(event: React.FormEvent<HTMLInputElement	HTMLTextAreaElement>, newValue?: string) => void

* Further localizations: 
  * goToToday	string	String to render for button to direct the user to today's date.
  *  placeholder	string	Placeholder for the input

* Rendering: 
  * renderCell	ShorthandRenderFunction	A render function to customize how cells are rendered in the Calendar.
  *  renderHeaderCell	ShorthandRenderFunction	A render function to customize how cells are rendered in the Calendar.